tests/test_lint.py

```diff
@@ -28,12 +28,37 @@
     'requests': '2.18.4',
 }
 
-class TestLint(unittest.TestCase):
 
-    def test_lint(self):
-        lint_report = komodo.lint(RELEASE, REPO)
-        self.assertEqual([], lint_report.dependencies)
-        self.assertEqual([], lint_report.versions)
+def test_release_name():
+    for valid in (
+        "bleeding-py27.yml",
+        "bleeding-py36.yml",
+        "/home/anyuser/komodo-releases/releases/bleeding-py27.yml",
+        "/home/anyuser/komodo/bleeding-py27.yml",
+        "/home/anyuser/komodo/2020.01.03-py36-rhel6.yml",
+        "/home/anyuser/komodo/2020.01.03-py27.yml",
+        "myrelease-py27.yml",
+        "myrelease-py36.yml",
+        "myrelease-py27-rhel6.yml",
+        "myrelease-py27-rhel7.yml",
+        "myrelease-py36-rhel6.yml",
+        "myrelease-py36-rhel7.yml"
+    ):
+        assert [] == lint.lint_release_name(valid)
+
+    for invalid in (
+        "bleeding",
+        "bleeding.yml",
+        "2020.01.01",
+        "2020.01.00.yml",
+        "/home/anyuser/komodo-releases/releases/2020.01.00.yml",
+        "bleeding-py27",
+        "bleeding-rhel6.yml"
+    ):
```

Comment:
This seems like a perfect case for [pytest.mark.parameterize ](https://docs.pytest.org/en/stable/parametrize.html)!

Reply:
Debateable. I kind of dislike `pytest.mark.parametrize` and this codebase uses it very unsuccessfully. I did change my test to use it though, so it's up to you.

Reply:
Yes, might not make much difference in this case, but think we can keep it as is now!